recipes/hmftools-orange/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "3.7.0" %}
-{% set sha256 = "5eefa9acb4040361ba4f56ec5670ccdcd0ac18e8b13e3bdbd6a6e6241104dfd1" %}
+{% set version = "3.7_beta" %}
```

Comment:
3.7_beta is before 3.7.0
Maybe you want 3.7.1_beta ?

Reply:
This is correct unfortunately - the 3.7_beta version is newer than 3.7.0. I'll talk with the Hartwig folks to see whether the 3.7_beta version can be adjusted with a major/minor increment

Reply:
ORANGE 3.7_beta has been re-released as ORANGE 3.7.1_beta and updated here accordingly, thanks for bringing this to attention!